src/perl/lib/wtsi_clarity/util/label.pm

```diff
@@ -5,6 +5,9 @@ use Carp;
 use Readonly;
 use DateTime;
 
+use wtsi_clarity::util::config;
+my $config = wtsi_clarity::util::config->new();
```

Comment:
Again, because it is a role for the epp scripts it will have the config attribute on $self.
So, hopefully we don't need these lines.